test/test_helper.rb

```diff
@@ -0,0 +1,34 @@
+require 'coveralls'
+Coveralls.wear!('rails')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.